src/app/shared/components/accordion/accordion.component.scss

```diff
@@ -0,0 +1,66 @@
+@import "src/assets/themes/variables";
+
+$edit-col-width: 108px;
+$margin-overlap: 6px;
+$border-width: 2px;
+$border-radius: 4px;
+
+.accordion-header-row {
+  border: $border-width solid $primary-blue;
+  border-radius: $border-radius;
+
+  @media screen and (min-width: 768px) {
+    .accordion-header {
+      flex: 1;
+      border-right: $border-width solid $primary-blue;
+    }
+  }
+
+  .edit-row {
+    border-bottom: $border-width solid $primary-blue;
+  }
+
+  .edit-column {
+    flex: 0 0 calc($edit-col-width - $border-width);
+  }
+
+  .edit-row,
+  .edit-column {
+    button.btn {
+      color: $primary-blue;
+
+      i {
+        border-bottom: 1px solid $primary-blue;
+      }
+    }
+  }
+}
+
+.accordion-body {
+  @media screen and (min-width: 768px) {
+    width: calc(100% - $edit-col-width);
+  }
+
+  border: $border-width solid $primary-blue;
+  border-radius: $border-radius;
+
+  // use a negative margin and solid background to overlap the
+  // accordion-header when the body is open, hiding the bottom
+  // border and rounded corner of the header
+  border-top: none;
+  border-top-left-radius: 0;
+  border-top-right-radius: 0;
+  margin-top: -$margin-overlap;
```

Comment:
Margins, padding, spacing, etc. can all also be handled by preexisting bootstrap classes to remain consistent with other components. We should rarely need to hardcode pixels. This `scss` works but likely will not play nice with future design changes (typ).

Reply:
I tried to use the `p-#` and `m-#` classes everywhere I could. But I had to do something a little bit hacky with negative margins and positive padding in this css to get the border working like the design without completely re-writing the accordions to use ng-bootstrap components. So I did end up setting a 6px negative margin and a 6px positive padding to explicitly hide the 4px rounded corner and the 2px border.